flake.nix

```diff
@@ -22,17 +30,43 @@
       url = "github:oxalica/rust-overlay";
       inputs.nixpkgs.follows = "nixpkgs";
     };
+
+    versions = {
+      url = "github:holochain/holochain/pr_holonix_on_flakes?dir=versions/0_1";
```

Comment:
can we change this to `/develop?dir=` after merging?

Reply:
actually we should change it now, thanks!